shared/actions/teams.tsx

```diff
@@ -178,7 +179,7 @@ const setWelcomeMessage = async (action: TeamsGen.SetWelcomeMessagePayload, logg
 }
 
 const getTeamRetentionPolicy = async (
-  state: TypedState,
+  _: TypedState,
```

Comment:
fyi if these don't need state you can just register it with `chainAction` and state won't be passed as the first param

Reply:
Refactoring those to use `s/chainAction2/chainAction/`. Is there a good solution to `chainGenerator`?